bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -1165,7 +1159,9 @@ public Color getBackground() {
 	/* Get the HDC for the device */
 	long hDC = device.internal_new_GC(null);
 	return applyUsingAnyHandle(imageHandle -> {
```

Comment:
I don't think we can use `anyHandle` here anymore, but need to use exactly the one based on the transparent pixel (probably usually the 100% one).